src/webview/utils/vscode-contexts.ts

```diff
@@ -0,0 +1,51 @@
+/********************************************************************************
+ * Copyright (C) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { WebviewIdMessageParticipant } from 'vscode-messenger-common';
+
+/**
+ * Custom data property used by VSCode to provide additional context info when opening a webview context menu
+ * The data property needs to represent a valid JSON object.  Data context properties up the parent chain are merged into the child context.
+ * This context info can be used to in `when` clauses from webview/context menu contributions and is accessible as argument of the corresponding command.
+ */
+export interface VscodeContext {
+    'data-vscode-context': string;
+}
+
+export interface AppVscodeContext {
+    messageParticipant: WebviewIdMessageParticipant;
+    visibleColumns: string;
+    showRadixPrefix: boolean;
+}
```

Comment:
Same as `WebviewMenuContext` in `webview-main.ts`. Perhaps it should be shared in the common area?